apps/website/blog/2024-12-01-maci-getting-started.md

```diff
@@ -0,0 +1,294 @@
+---
+slug: Getting Started MACI
+title: Getting Started with MACI
+description: Guide to use MACI
+authors:
+  name: Crisgarner
+  title: MACI team contributor
+  url: https://x.com/crisgarner
+  image_url: https://avatars.githubusercontent.com/u/578688?v=4
+tags: [voting, security, anonymity, roadmap, community, MACI, development]
+excerpt: "Get Started with MACI"
+---
+
+Hey folks, we are excited to start a series of tutorials on how to get started with all things MACI, this guide focuses on installing MACI and how to deploy the contracts to a testnet.
```

Comment:
Hey folks! We’re thrilled to kick off a series of tutorials covering everything you need to know about MACI. This guide will focus on installing MACI and deploying the contracts to a testnet. Let’s get started!
(by ChatGPT)
(for me a non-native speaker, I think this is more understandable)